components/financial_assistance/app/views/financial_assistance/applications/_faa_progress_options.html.erb

```diff
@@ -0,0 +1,26 @@
+<% step = (defined? step) ? step.to_i : 1 %>
+
+<% content_for(:step) { step } %>
+<% content_for(:show_exit_button) { "true" } %>
+<% content_for(:show_previous_button) { "false" } %>
+<% content_for(:show_account_button) { EnrollRegistry.feature_enabled?(:back_to_account_all_shop) } %>
```

Comment:
should this be a string? I know we talked about how content_for only cares about strings, I think `feature_enabled?` returns a bool so this may be a problem

Reply:
fixing